staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go

```diff
@@ -26,6 +26,12 @@ const (
 	LabelOS   = "beta.kubernetes.io/os"
 	LabelArch = "beta.kubernetes.io/arch"
 
+	// When kubelet is started with the "external" cloud provider, then
+	// it sets this label on the node to denote an ip address set from the
+	// cmd line flag. This ip is verified with the cloudprovider as valid by
+	// the cloud-controller-manager
+	LabelProvidedIPAddr = "beta.kubernetes.io/provided-node-ip"
```

Comment:
This is confusing. The explanation of Annotation vs Labels is provided in the online docs (https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/). Why are we going against that? Care to explain @luxas?